.travis.yml

```diff
@@ -0,0 +1,13 @@
+---
+cache: pip
+language: python
+
+python:
+  - "3.7"
+  - "3.7-dev"  # 3.7 development branch
+
+install:
+- pip install .[all]
+
+script:
+- pytest
```

Comment:
I think it's missing the `pycodestyle`? It was mentioned in the issue description in #8 . Could you add that too, please, @ytturi ?

Reply:
Done 👍 
Sorry it was the first point and I totally forgot about it 😢

Reply:
No worries, I almost forgot it too. Only remembered because I was using `pycodestyle` moments before reviewing it 🙂

Reply:
By the way, if you update the file because of the newline, could you switch the order, and run `pycodestyle` first? Just because it's normally faster than the tests, so it is better that it fails first as well, so we don't bother waiting for the tests (which are super fast now, but as we add more they may get slower).

Thanks!